pokemon/src/main/java/gamarket/EncounterGUI.java

```diff
@@ -1,125 +1,1075 @@
+        hp1 = new Button(this.updateHP1());
+        hp2 = new Button(this.updateHP2());
+ 
+       // Set up the container to hold the player's pokemon and wild pokemon images and info bars. 
+        playerSide = new GridPane();     
+        wildSide = new GridPane();
+        playerSide.add(playerPoke, 0, 2);  
+        playerSide.add(hp1, 0, 0);       
+        wildSide.add(enemyPoke, 0, 2);
+        wildSide.add(hp2, 0, 0);
+ 
+        // Adding the pokemon to the GUI 
+        poke.add(wildSide, 1, 1);     
+        poke.add(playerSide, 0, 2);     
+        poke.setHgap(425);
+        poke.setVgap(145);
+        poke.setPadding(new Insets(15, 0, 0, 550)); 
+ 
+        // Set the maximum height for the move buttons. 
+        move1.setMaxHeight(70);
+        move2.setMaxHeight(70);
+        move3.setMaxHeight(70);
+        move4.setMaxHeight(70);
+ 
+        // Create the big textbox to update real-time text upon the pokemon wild encounter. 
+        StackPane mySP = new StackPane(); 
+        box = new Button();
+        box.setStyle("-fx-pref-width: 1350px; " + "-fx-pref-height: 250px;"
+                    + "-fx-border: none; "  + "-fx-background-color: gray;" + "-fx-font-size: 32px;"
+                    + "-fx-text-align: center;" + "-fx-text-base-color: white;");
+        mySP.getChildren().add(box);
+        box.setPadding(new Insets(125, 0,0,0));
+        mySP.setPadding(new Insets(185, 0,0,0));            // Responsible for text box positioning. 
+ 
+        // The container to hold our text. 
+        Group textbox = new Group();
+        text.setStroke(Color.BLACK);
+        text.setFill(Color.WHITE);
+        textbox.getChildren().add(text);
+        
+        // Add the textbox that contains our text to the wild pokemon encounter GUI. 
+        textlols = new GridPane();
+        textlols.getChildren().add(textbox);
+        textlols.setPadding(new Insets(25, 0, 0, 110));
+ 
+        mySP.getChildren().add(textlols);
+        battleMenu.setPadding(new Insets(30, 0, 0, 230));
+        fightMenu.setPadding(new Insets(50, 0, 0, 185));
+        
+        // Add everything to the main GUI container: the pokemons, menu, and text box. 
+        gp.add(poke, 0, 0);
+        gp.add(menu, 1, 1);
+        gp.add(mySP, 0, 1);
+        gp.setStyle("-fx-alignment: center-right;");
+        
+        // Style them health bars and add them to the main GridPane GUI 'gp'.
+        StackPane playerPokeHealthBar = new StackPane();        // The player pokemon. 
+        playerPokeHealthBar.getChildren().add(hp2);
+        gp.add(playerPokeHealthBar, 1, 0);
+        playerPokeHealthBar.setPadding(new Insets(0, 0, 532, 300));
+        hp2.setStyle("-fx-background-radius: 300px;" + "-fx-pref-width: 300px; " + "-fx-border: none; "
+            + "-fx-background-color: white;" + "-fx-border-radius: 300px;" + "-fx-border-width: 7px;"
+            + "-fx-font-size: 24px;" + "-fx-text-align: center;" + "-fx-text-base-color: rgb(0,0,0);");

```

Comment:
the styles for hp2 and hp1 are the same value should be stored as a variable hpStyle